services/mailservice/mail.go

```diff
@@ -191,14 +190,14 @@ func TestConnection(config *model.Config) {
 
 	conn, err1 := ConnectToSMTPServer(config)
 	if err1 != nil {
-		mlog.Error(fmt.Sprintf("SMTP server settings do not appear to be configured properly err=%v details=%v", utils.T(err1.Message), err1.DetailedError))
+		mlog.Error("SMTP server settings do not appear to be configured properly", mlog.String("error_message", utils.T(err1.Message)), mlog.String("error_details", err1.DetailedError))
```

Comment:
It's unfortunate that we're adding a new structured field name here: would we consider just `mlog.Err(err)` to make it easy on those consuming the structured logs?

Reply:
Yes, I think is safe because is used to build the error string in `Error()`, so lets change it to use `mlog.Err(err)`

Reply:
Done